tests.sh

```diff
@@ -7,7 +7,7 @@ function test_eval() {
   input=$1
   expected_output1="ans = "$2
   expected_output2="ans = "$3
-  error_output=""$2
+  error_output=$2" (Errc"
```

Comment:
interestin! When I saw it in the previous PR I was with this feeling, that this pattern was way to general.